content/sensu-go/6.0/release-notes.md

```diff
@@ -127,6 +128,20 @@ See the [supported platforms][165] page for a complete list of Sensu’s support
 With this fix, labels and annotations are always defined (although they may be empty).
 - Fixed the log entry field for the check's name in schedulerd.
 
+## 5.21.3 release notes
+
+**October 14, 2020** &mdash; The latest release of Sensu Go, version 5.21.3, is now available for download.
```

Comment:
@cwjohnston should we say something to qualify this as the latest 5.x release or something?

Reply:
```suggestion
**October 14, 2020** &mdash; The latest release of Sensu Go 5, version 5.21.3, is now available for download.
```